client/src/app/tabs/bpmn-shared/modeler/features/apply-default-templates/applyDefaultTemplates.js

```diff
@@ -9,6 +9,11 @@
  */
 
 export default function applyDefaultTemplates(elementRegistry, elementTemplates, commandStack, changeTemplateCommand) {
```

Comment:
It's a bit odd that we allow the command name to be configured but always assume the same structure of the command context. 🤔

Reply:
Yes. This is a code smell to start with, but I did not attempt to clean it up at this stage.